framework.py

```diff
@@ -32,11 +27,10 @@ def poc():
     Sizes=["S", "M", "L"]
     return render_template("POC page.html", store_name=Store_name, categories=Categories, items=Items, sizes=Sizes, title=title)
 
-@app.route("/add_to_shopping_bag")
-def add_to_shopping_bag():
-    pass
-
-
+@app.route("/Cart", methods=["POST"])

```

Comment:
it seems that this route returns only the cart page, so it should be a GET method. 
you should use POST method when a route should create something. for example, a route that creates a new user.